modules/oadp-restic-issues.adoc

```diff
@@ -99,13 +99,43 @@ $ oc get all -n <namespace>
 [id="restic-backup-cannot-be-recreated-after-s3-bucket-emptied_{context}"]
 == Restic Backup CR cannot be recreated after bucket is emptied
 
-If you create a Restic `Backup` CR for a namespace, empty the S3 bucket, and then recreate the `Backup` CR for the same namespace, the recreated `Backup` CR fails.
+If you create a Restic `Backup` CR for a namespace, empty the object storage/bucket, and then re-create the `Backup` CR for the same namespace, the re-created `Backup` CR fails.
 
-The `velero` pod log displays the error message, `msg="Error checking repository for stale locks"`.
+The `velero` pod log displays the following error message: `stderr=Fatal: unable to open config file: Stat: The specified key does not exist.\nIs there a repository at the following location?`
 
 .Cause
 
-Velero does not create the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted on object storage. See (link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421]) for details.
+Velero does not re-create or update the Restic repository from the `ResticRepository` manifest if the Restic directories are deleted from object storage. For more information, see link:https://github.com/vmware-tanzu/velero/issues/4421[Velero issue 4421].
+
+.Solution
+
+* Remove the related Resitc repository from the namespace by running the following command:
++
+[source,terminal]
+----
+oc delete resticrepository openshift-adp <name-of-the-restic-repository>
+----
++
+
+In the following error log, `mysql-persistent` is the problematic Restic repository. The name of the repsoitory appears in talics for clarity.
```

Comment:
@RichardHoch I highly recommend a spell checker for your editor. If it's VSCode, I use this one: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
@jboxman-rh I made the changes you suggested and reactivated my spell checker, which was deactivated for reasons unknown to me. Please recheck this PR.